### PR TITLE
fix(nuxi): show base url in terminal message

### PR DESCRIPTION
**Pull request by antfu**
### 🔗 Linked issue

Requires https://github.com/unjs/listhen/pull/28

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

When we configure `app.baseURL`, the terminal still shows the root URL like `http://localhost:3000/`.

This PR requires upstream https://github.com/unjs/listhen/pull/28

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxi/src/commands/dev.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export default defineNuxtCommand({
       port: args.port || args.p || process.env.NUXT_PORT,
       hostname: args.host || args.h || process.env.NUXT_HOST,
       https: Boolean(args.https),
+      baseURL: args.baseURL || process.env.NUXT_BASE_URL,
```
Comment (pi0) [inline packages/nuxi/src/commands/dev.ts:41]: If we are to introduce this new flag, I guess we have to pass from args (`--baseUrl`) to the options (`loadNuxt`) as well.

Note: We might rename it to `--config-app-baseURL`. I'm not sure about final convention, but at some point, allowing to override every Nuxt config using env and args, could reduce the chance to deprecate arg. But nothing important.

**Comment by pi0** [comment]
Considering upstream merged, what happens if user is setting base URL directly in `nuxt.config`? I guess first banner would show without and then subsequent reloads add base with override of `showURL`?

**Comment by antfu** [comment]
Are you ok with that we disable `showUrl` option in listen and call it manually after the config resolve?

**Comment by pi0** [comment]
@antfu Any update on this?

**Comment by antfu** [comment]
Shall be ready now.

**Review by pi0 — APPROVED** [review]

**Review by pi0 — APPROVED** [review]

**Pull request merged**